### PR TITLE
Bug/editor crash

### PR DESCRIPTION
**Pull request by feruzm**
Linting fixes
precommit fix
and other bug fixes, editor, wallet icon, background, etc. fixes

To Test, don't forget to run `yarn`

### CODE REVIEW

File: src/components/leaderboard/view/leaderboardView.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ class LeaderboardView extends PureComponent {
           options={VALUE.map(val => intl.formatMessage({ id: `leaderboard.${val}` }))}
           selectedOptionIndex={selectedIndex}
           defaultText={intl.formatMessage({ id: `leaderboard.${VALUE[0]}` })}
-          onDropdownSelect={selectedIndex =>
-            fetchLeaderBoard(FILTER_OPTIONS[selectedIndex], selectedIndex)
+          onDropdownSelect={selectedIndexM =>
```
Comment (ue) [inline src/components/leaderboard/view/leaderboardView.js]: what is M meaning?

File: src/components/editorElements/titleArea/view/titleAreaView.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ export default class TitleAreaView extends Component {
             this.setState({ height: event.nativeEvent.contentSize.height });
           }}
           autoFocus={autoFocus}
-          onChangeText={text => this._handleOnChange(text)}
+          onChangeText={textT => this._handleOnChange(textT)}
```
Comment (ue) [inline src/components/editorElements/titleArea/view/titleAreaView.js]: what is T meanin

Comment (feruzm) [reply src/components/editorElements/titleArea/view/titleAreaView.js]: It is just making variable different than already used variable name

File: src/components/leaderboard/view/leaderboardView.js (hunk 1 above)
Comment (feruzm) [reply src/components/leaderboard/view/leaderboardView.js]: It is just making variable different than already used variable name

File: src/components/editorElements/titleArea/view/titleAreaView.js (hunk 2 above)
Comment (feruzm) [reply src/components/editorElements/titleArea/view/titleAreaView.js]: Linting complained so I tried to decrease warnings and slowly we can get rid of all warnings from linting

**Comment by feruzm** [comment]
@ue is everything else look ok to you so we can merge?

**Comment by feruzm** [comment]
If there is something that needs urgent changing please do in separate PR 😉

**Pull request merged**